dissect/target/helpers/record.py

```diff
@@ -136,6 +136,25 @@ def DynamicDescriptor(types):  # noqa
     ],
 )
 
+WindowsKeyboardRecord = TargetRecordDescriptor(
```

Comment:
I suggest to keep these records local (so in the file that uses them), instead of in this general location